docs/guide/features.md

```diff
@@ -30,7 +30,17 @@ ESM first, top level await
 
 ## Threads
 
-Workers multi-threading via [tinypool](https://github.com/Aslemammad/tinypool) (a lightweight fork of [Piscina](https://github.com/piscinajs/piscina))
+Workers multi-threading via [tinypool](https://github.com/Aslemammad/tinypool) (a lightweight fork of [Piscina](https://github.com/piscinajs/piscina)), allowing tests to run simultaneously. Threads are enabled by default Vitest, and can be disabled passing `--no-threads` in the CLI.
+
+ ```bash
+$ vitest --no-threads
```

Comment:
I think we should remove the code blocks, as people will focus on them. The option in the text is enough here

Reply:
done!